src/thread.rs

```diff
@@ -242,7 +242,20 @@ impl SBThread {
     }
 
     #[allow(missing_docs)]
-    pub fn step_into(
+    pub fn step_into(&self, stop_other_threads: RunMode) -> Result<(), SBError> {
+        let error = SBError::default();
```

Comment:
This error isn’t used here… maybe could call the other version and pass defaults for target and line number that let them be ignored?